src/main/java/com/ifmo/epampractice/daoimpl/DrivingLicenseImpl.java

```diff
@@ -0,0 +1,113 @@
+package com.ifmo.epampractice.daoimpl;
+
+import com.ifmo.epampractice.dao.DBConnectorInterface;
+import com.ifmo.epampractice.dao.DBConnectorPostgres;
+import com.ifmo.epampractice.dao.DrivingLicenseDAO;
+import com.ifmo.epampractice.entity.DrivingLicenseEntity;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DrivingLicenseImpl implements DrivingLicenseDAO {
+
+    private static final String GET_ALL_QUERY = "SELECT * FROM driving_license";
+    private static final String GET_QUERY = "SELECT * FROM driving_license WHERE ID = ?";
+    private static final String UPDATE_QUERY = "UPDATE driving_license SET (Issue_date, Expiration_date, Serial_number) = (?, ?, ?) WHERE ID = ?";
+    private static final String DELETE_QUERY = "DELETE FROM driving_license WHERE ID = ?";
+    private static final String SAVE_QUERY = "INSERT INTO driving_license (Issue_date, Expiration_date, Serial_number) VALUES (?, ?, ?)";
+
+    private static final Logger LOG = LogManager.getLogger(DrivingLicenseImpl.class);
+    private DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+
+    @Override
+    public List<DrivingLicenseEntity> getAll() {
+        List<DrivingLicenseEntity> drivingLicenseEntityArrayList = new ArrayList<>();
+
+        try (Connection connection = dbConnector.getConnection();
+             Statement statement = connection.createStatement(
+                     ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY);
+             ResultSet resultSet = statement.executeQuery(GET_ALL_QUERY)) {
```

Comment:
Statement and ResultSet should be closed after usage

Reply:
It'll be auto closed

Reply:
Didn't see all these statements are inside try-with-resources